### PR TITLE
Fix processors configuration in windows and o365 packages

### PR DESCRIPTION
**Pull request by adriansr**
## What does this PR do?

Fixes the _processors_ configuration option for `windows` and `o365` packages.

Before this change, trying to add custom processors to any of their data streams resulted in the following error:

image

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- ~~[ ] If I'm introducing a new feature, I have modified the Kibana version constraint in my package's `manifest.yml` file to point to the latest Elastic stack release (e.g. `^7.13.0`).~~

### CODE REVIEW

File: packages/windows/data_stream/forwarded/agent/stream/httpjson.yml.hbs [hunk 1]
```diff
@@ -59,43 +59,43 @@ tags:
...
+    fail_on_error: false
+- drop_fields:
+    fields: json
+- decode_xml_wineventlog:
+    field: event.original
+    target_field: winlog
+    ignore_missing: true
+    ignore_failure: true
+    map_ecs_fields: true
 {{#if processors.length}}
```
Comment (andrewkroh) [inline packages/windows/data_stream/forwarded/agent/stream/httpjson.yml.hbs:99]: This is using `if processors.length` while others are using `if processors`. Any idea why? And could we change this one to be consistent with the others without impacting behavior.

**Review by andrewkroh — APPROVED** [review]
LGTM, but I did have a question.

File: packages/windows/data_stream/forwarded/agent/stream/httpjson.yml.hbs (hunk 1 above)
Comment (adriansr) [reply packages/windows/data_stream/forwarded/agent/stream/httpjson.yml.hbs:99]: I see we have 300 usages of `if processors` and 10 for `processors.length`. I have no idea what the difference is but both worked fine in my tests.

For now I think it's better to leave this PR as is, each package using its original condition, and evaluate separately if we need to change all packages to use the better of the two kinds of conditions.

**Comment by adriansr** [comment]
Fixes the processors configuration option for windows and o365 packages, it
was failing due to bad indentation.

**Pull request merged**